pkg/wireguard/agent/agent.go

```diff
@@ -121,7 +127,7 @@ func (a *Agent) Init() error {
 		return err
```

Comment:
Line 107: `sysctl.Disable(fmt.Sprintf("net.ipv4.conf.%s.rp_filter", types.IfaceName))`

Reply:
I will leave this unchanged to avoid conflicts with this line in the upstream branch (it's completely unrelated to this PR). @brb feel free to adapt that in your commit.

Reply:
https://github.com/cilium/cilium/pull/15565/commits/6148cc24914a4edce990839f87d7611674ebde67 addressed this.